tests/page-objects/default/reports/reports.wdio.page.js

```diff
@@ -335,87 +340,79 @@ const getDetailReportRowContent = async (row) => {
 
 const getOpenReportInfo = async () => {
   return {
-    patientName: await getElementText(patientName()),
-    reportName: await getElementText(reportName()),
-    senderName: await getElementText(senderName()),
-    senderPhone: await getElementText(senderPhone()),
-    lineage: await getElementText(lineage()),
-    relativeDate: await getElementText(relativeDate()),
+    patientName: await getElementText(rightPanelSelectors.patientName()),
+    reportName: await getElementText(rightPanelSelectors.reportName()),
+    senderName: await getElementText(rightPanelSelectors.senderName()),
+    senderPhone: await getElementText(rightPanelSelectors.senderPhone()),
+    lineage: await getElementText(rightPanelSelectors.lineage()),
+    relativeDate: await getElementText(rightPanelSelectors.relativeDate()),
   };
 };
 
 const openSelectedReport = async (listElement) => {
   await listElement.click();
 };
 
-const resetFilter = async () => {
-  await openSidebarFilter();
-  await (await filterResetBtn()).waitForDisplayed();
-  await (await filterResetBtn()).click();
-  await closeSidebarFilter();
-  await searchElements.clearSearch();
-};
-
 const openReport = async (reportId) => {
-  await (await firstReport()).waitForDisplayed();
-  const reportListItem = await reportByUUID(reportId);
+  await (await leftPanelSelectors.firstReport()).waitForDisplayed();
+  const reportListItem = await leftPanelSelectors.reportByUUID(reportId);
   await reportListItem.waitForClickable();
   await reportListItem.click();
-  await reportBodyDetails().waitForDisplayed();
+  await rightPanelSelectors.reportBodyDetails().waitForDisplayed();
 };
 
 const editReport = async () => {
   await commonElements.openMoreOptionsMenu();
-  await (await editReportButton()).waitForClickable();
-  await (await editReportButton()).click();
+  await (await kebabMenuSelectors.edit()).waitForClickable();
+  await (await kebabMenuSelectors.edit()).click();
 };
 
 const fieldByIndex = async (index) => {
-  return await (await $(`${REPORT_BODY_DETAILS_SELECTOR} li:nth-child(${index}) p`)).getText();
+  return await (await $(`${REPORT_BODY_DETAILS} li:nth-child(${index}) p`)).getText();
 };
 
 const exportReports = async () => {
   await commonElements.openMoreOptionsMenu();
-  await (await exportButton()).waitForClickable();
-  await (await exportButton()).click();
+  await (await kebabMenuSelectors.export()).waitForClickable();
+  await (await kebabMenuSelectors.export()).click();
 };
 
 const deleteReport = async () => {
   await commonElements.openMoreOptionsMenu();
-  await (await deleteButton()).waitForClickable();
-  await (await deleteButton()).click();
+  await (await kebabMenuSelectors.delete()).waitForClickable();
+  await (await kebabMenuSelectors.delete()).click();
 };
```

Comment:
Same here `editReport`, `exportReports` and `deleteReport` has duplicate codes.
We could improve this by creating something like this:
```
const performMenuAction = async (actionSelector) => {
  await commonElements.openMoreOptionsMenu();
  const actionElement = await actionSelector();
  await actionElement.waitForClickable();
  await actionElement.click();
};
```
And again, no simplify refactor, define:
```
const editReport = async () => {
  await performMenuAction(kebabMenuSelectors.edit);
};

const exportReports = async () => {
  await performMenuAction(kebabMenuSelectors.export);
};

const deleteReport = async () => {
  await performMenuAction(kebabMenuSelectors.delete);
};
```